### PR TITLE
ISS-121464: updated the iOS native sdk pod version to 6.0.9 with new always change

### PR DESCRIPTION
**Pull request by dhananjayj-dev**
github pr template
## Description :pencil2:
Updated the plugin  with new iOS native sdk method support 
public class func stopRecording(resetAppSessionId: Bool)

Created a new ASI (App Session ID) on explicit calls to stopRecording() calls by client—IndusInd

A new App Session ID on every stopRecording will be useful in the client's heatmap generatio

We will have another variant of stopRecording() public function, so existing clients won’t be impacted:

`public class func stopRecording(resetAppSessionId: Bool)`

Based on this flag, we will create a new ASI (App Session ID)

So client, (IndusInd) will have to call:
`UserExperior.stopRecording(resetAppSessionId: true)`

## Related links :round_pushpin:
ISS-121464

### CODE REVIEW

**Review by zeeshan-devrev — CHANGES_REQUESTED** [review]
Please remove .idea folder, and other temporary files like .DS_Store, and only push the necessary changes

**Comment by dhananjayj-dev** [comment]
> Please remove .idea folder, and other temporary files like .DS_Store, and only push the necessary changes

Removed the same

**Review by dhananjayj-dev — COMMENTED** [review]
Done!

File: userexperior-cordova-plugin/src/ios/CDVUserExperior.m [hunk 1]
```diff
@@ -60,8 +60,16 @@ - (void)pauseRecording:(CDVInvokedUrlCommand*)command {
...
 
 - (void)stopRecording:(CDVInvokedUrlCommand*)command {
-    [UserExperior stopRecording];
+    if (command.arguments.count == 0) {
+        // No arguments provided, call the function without resetAppSessionID
+        [UserExperior stopRecording];
+    } else {
+        // If any argument (boolean) provided, call the function with resetAppSessionID
+        BOOL resetAppSessionID = [[command.arguments objectAtIndex:0] boolValue];
+        [UserExperior stopRecordingWithResetSession:resetAppSessionID];
```
Comment (zeeshan-devrev) [inline userexperior-cordova-plugin/src/ios/CDVUserExperior.m]: Is this correct function name, as I couldn't find this function in the native SDK here:

[UserExperior-Development/UserExperiorSDK/UserExperiorSDK/public/UserExperior.swift](https://github.com/devrev/ue-ios-sdk/pull/1/files#diff-4cd87540b339727525a7d2a4dcaf2ce827ef51f4deda25080dc0d9e5e9881bef)

File: userexperior-cordova-plugin/src/ios/CDVUserExperior.m
Comment (zeeshan-devrev) [inline userexperior-cordova-plugin/src/ios/CDVUserExperior.m:1]: Do you have changes to `CDVUserExperior.h` that have not been pushed?

File: userexperior-cordova-plugin/src/ios/CDVUserExperior.m (hunk 1 above)
Comment (dhananjayj-dev) [reply userexperior-cordova-plugin/src/ios/CDVUserExperior.m]: this was slightly different in obj c. I updated it to the correct one.

**Review by zeeshan-devrev — APPROVED** [review]

**Pull request merged**